internal/core/metadata/application/device.go

```diff
@@ -165,6 +189,14 @@ func PatchDevice(dto dtos.UpdateDevice, ctx context.Context, dic *di.Container)
 		go updateDeviceCallback(ctx, dic, oldServiceName, device)
 	}
 	go updateDeviceCallback(ctx, dic, device.ServiceName, device)
+
+	go func() {
+		if err := publishDeviceSystemEvent(common.DeviceSystemEventActionUpdate, device, ctx, lc, dic); err != nil {
+			// Don't fail request if couldn't publish, just log it. Error message already has context.
```

Comment:
since it's asnyc, we might be able to remove this comment

Reply:
No longer needed